src/uct/api/uct.h

```diff
@@ -2306,11 +2309,15 @@ UCT_INLINE_API ucs_status_t uct_iface_tag_recv_cancel(uct_iface_h iface,
  * Notify the transport that it should actively progress communications during
  * @ref uct_worker_progress().
  *
- * When the interface is created, its progress is enabled.
+ * When the interface is created, its progress is initially disabled.
```

Comment:
The corresponding comment for uct_iface_progress_disable needs to be changed to similarly indicate that progress is disabled when the interface is created.

Reply:
fixed